ui/src/main/js/page/stocks/index.js

```diff
@@ -0,0 +1,180 @@
+'use strict';
+
+import React from 'react';
+import StockList from '../../component/stock-list/index.js';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/modernmaster/forecaster/pullRequest?prid=7238741)
```suggestion
import StockList from "../../component/stock-list/index.js";
```